code/modules/antagonists/eldritch_cult/eldritch_book.dm

```diff
@@ -6,70 +6,70 @@
 	w_class = WEIGHT_CLASS_SMALL
 	///Last person that touched this
 	var/mob/living/last_user
-	///how many charges do we have?
-	var/charge = 1
-	///Where we cannot create the rune?
-	var/static/list/blacklisted_turfs = typecacheof(list(/turf/closed,/turf/open/space,/turf/open/lava))
 	///Is it in use?
 	var/in_use = FALSE
+	var/charge = 0
 
 /obj/item/forbidden_book/Destroy()
 	last_user = null
 	. = ..()
 
-
 /obj/item/forbidden_book/examine(mob/user)
 	. = ..()
 	if(!IS_HERETIC(user))
 		return
-	. += "The Tome holds [charge] charges."
-	. += "Use it on the floor to create a transmutation rune, used to perform rituals."
-	. += "Hit an influence in the black part with it to gain a charge."
-	. += "Hit a transmutation rune to destroy it."
+	//revise
+	//. += "The Tome holds [charge] charges."
+	//. += "Use it on the floor to create a transmutation rune, used to perform rituals."
+	//. += "Hit an influence in the black part with it to gain a charge."
+	//. += "Hit a transmutation rune to destroy it."
+	. += "Any mortal that reads this book will gain fascination. Baptise them with your Mansus Grasp to turn them into your disciples."
+	var/datum/antagonist/heretic/cultie = user.mind.has_antag_datum(/datum/antagonist/heretic)
+	. +=  "You have earned [cultie.get_favor_left()] favor for your deeds."
+	for (var/EK in cultie.get_all_knowledge())
+		var/datum/eldritch_knowledge/known = EK
+		if (istype(known))
+			. +=  known.desc
 
 /obj/item/forbidden_book/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
-	. = ..()
 	if(!proximity_flag || !IS_HERETIC(user) || in_use)
 		return
+	var/datum/antagonist/heretic/cultie = user.mind.has_antag_datum(/datum/antagonist/heretic)
+	var/mob/living/carbon/human/victim = target
 	in_use = TRUE
-	if(istype(target,/obj/effect/eldritch))
-		remove_rune(target,user)
+	if (istype(victim) && victim.stat != DEAD && cultie.get_knowledge(/datum/eldritch_knowledge/dreamgate))
+		to_chat(user,"<span class='warning'>You open the [src] and start assaulting the mind of [victim]!</span>")
+		icon_state = "book_open"
+		flick("book_opening",src)
+		if (do_after(user,10 SECONDS,victim))
+			user.whisper("That is not dead which can eternal lie...", language = /datum/language/common)
+			var/dream_text = pick ("a hooded figurine","dead bodies, as far as the eye can see","whispering","opens a third eye","grows tentacles", "the monster of a thousand hands","beautiful creatures made out of of flesh and bone","a book... written in blood and bile")
+			to_chat(victim, "<span class='warning'>... [dream_text]...</span>")
+			if (do_after(user,10 SECONDS,victim) && !QDELETED(victim) && victim.stat != DEAD && victim.IsSleeping())
```

Comment:
The victim has to be asleep? This seems like it is going to be almost impossible to recruit anyone unless there is a heretic ability to put people to sleep.

Reply:
I was thinking I replace this one with mask of madness entirely (no point of it since you can already convert on the go)

Might as well port some of the other changes here (the new rust spell, those rune thingies...)